scoresheets/CleanTable.tex

```diff
@@ -2,18 +2,17 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[5]{60}{Picking tableware and cutlery for transportation to the dishwasher}
-	\scoreitem[5]{180}{Placing the tableware and cutlery inside the dishwasher}
+	\scoreitem[5]{240}{Placing the tableware and cutlery inside the dishwasher}
 
 	\scoreheading{Bonus Rewards}
 	\scoreitem{200}{Opening the dishwasher door}
 	\scoreitem{100}{Pulling out the dishwasher rack}
-	\scoreitem[5]{40}{Placing an item correctly in the dishwasher}
-	\scoreitem{300}{Placing the dishwasher tab inside the dishwasher}
+	\scoreitem{200}{Placing all items correctly in the dishwasher}
+	\scoreitem{300}{Taking and placing the dishwasher tab inside the dishwasher}
 
 	\scoreheading{Deus Ex Machina Penalties}
-	\penaltyitem[5]{25}{Pointing at an object or telling the robot where an object is}
-	\penaltyitem[5]{60}{Handing an object over to the robot}
+	\penaltyitem[5]{100}{Pointing at an object or telling the robot where an object is}
+	\penaltyitem[5]{140}{Handing an object over to the robot}
```

Comment:
I think these are the biggest ways we can improve the challenge. Increasing the penalty for handover. 

I am assuming the penalty for handing over an object is cumulative with not receiving points for pickup.

Reply:
True. Adding "per delivered object" should work.